.github/workflows/ci.yml

```diff
@@ -25,17 +25,15 @@ jobs:
           java-package: jdk
           architecture: x64
 
-      - name: Clean
-        run: sbt clean
-
       - name: Compile
         run: sbt +compile
 
-      - name: Test - Compile 
-        run: sbt +test:compile
+      - name: Test
+        run: sbt test:compile
 
-      # - name: Test
-      #   run: sbt +test
```

Comment:
We should not leave commented out code especially when it is about ensuring some level of quality.
I hope this is just transient while work is in progress otherwise I would like to see a ticket where this will be fixed.

Reply:
+1

Reply:
These are commented in `main` as well.